### ISSUE TITLE
FBT003: Ignore by default django.db.models.Value

### ISSUE THREAD

**Issue by 9128305** [astral-sh/ruff#10356]
Hi, there is a common usage of `Value(False)` | `Value(True)`  in [Django](https://docs.djangoproject.com/en/5.0/ref/models/expressions/#value-expressions) projects. Ruff does not provide any configuration to extend the FBT003 default whitelist. So my request would be 1) Ignore `Value` by default like https://github.com/astral-sh/ruff/issues/6711 2) settings to extend FBT003 whitelist.

```python
from django.db.models import Case, Q, Value, When
...
qs.annotate(
    is_foo_or_bar=Case(
        When(Q(is_foo=True) | Q(is_bar=True)),
        then=Value(True),  # noqa: FBT003
    ),
    default=Value(False),  # noqa: FBT003
)
```

**Comment by zanieb** [comment]
An allow-list for this rule does seems reasonable.

**Comment by ngnpope** [comment]
@9128305 It should be possible to use `then=True` and `default=False` directly without wrapping with `Value()`, and that goes for most basic types, not just booleans. The exception is where strings are used, as `then="string"` would be interpreted as `then=F("string")`, i.e. a column name, and so `then=Value("string")` is required explicitly.
